packages/mobile/src/escrow/EscrowedPaymentListItem.tsx

```diff
@@ -38,7 +38,7 @@ function EscrowedPaymentListItem({ payment }: Props) {
         }),
       })
     } catch (error) {
-      Logger.error(TAG, `Error sending reminder to ${recipient.e164PhoneNumber}`, error)
+      Logger.warn(TAG, `Error sending reminder to ${recipient.e164PhoneNumber}`, error)
```

Comment:
This is interesting to capture as well.

Reply:
Hmm, the phone number is considered PII, right? I just removed it from the log, don't think it adds a ton of value having it anyways.

Reply:
Yes, I meant the error itself, we can redact the phone number.